ci-operator/step-registry/openshift-qe/berserker-load/openshift-qe-berserker-load-commands.sh

```
@@ -0,0 +1,103 @@
+        fi
+    fi
+fi
+
+REPO_URL="${E2E_REPO:-https://github.com/cloud-bulldozer/e2e-benchmarking}";
+if [ "$E2E_VERSION" == "default" ]; then
+  LATEST_TAG=$(curl -s "https://api.github.com/repos/cloud-bulldozer/e2e-benchmarking/releases/latest" | jq -r '.tag_name');
+  TAG_OPTION="--branch ${LATEST_TAG}"
+else
+  TAG_OPTION="--branch ${E2E_VERSION}"
+fi
+git clone $REPO_URL $TAG_OPTION --depth 1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)